spec/integration/util/windows/security_spec.rb

```diff
@@ -388,8 +387,8 @@ def grant_everyone_full_access(path)
 
         describe "#owner=" do
           it "should accept a user sid" do
-            winsec.set_owner(sids[:admin], path)
-            expect(winsec.get_owner(path)).to eq(sids[:admin])
+            winsec.set_owner(sids[:current_user], path)
+            expect(winsec.get_owner(path)).to eq(sids[:current_user])
```

Comment:
This test category seems to be designed to test the Administrator changing permissions away from :guest, so I left that part and added tests to confirm that ownership was actually set to guest.